components/ws-manager/pkg/manager/create.go

```diff
@@ -400,6 +422,32 @@ func (m *Manager) createDefiniteWorkspacePod(startContext *startWorkspaceContext
 		}
 	}
 
+	if req.Type == api.WorkspaceType_IMAGEBUILD {
+		pod.Spec.Volumes = append(pod.Spec.Volumes, corev1.Volume{
+			Name: "gitpod-ca-certificate",
+			VolumeSource: corev1.VolumeSource{
+				Secret: &corev1.SecretVolumeSource{
+					SecretName: "ws-manager-tls",
```

Comment:
this still does not sit right. I understand we're just mounting the `ca.crt`, yet, considering the sensible nature of the ws-manager TLS cert, I'm wondering if there isn't another way.